xml/System/Memory`1.xml

```diff
@@ -731,7 +731,9 @@ The garbage collector will not move the memory until the returned <xref:System.B
       <Docs>
         <summary>Returns a span from the current instance.</summary>
         <value>A span created from the current <see cref="T:System.Memory`1" /> object.</value>
-        <remarks>To be added.</remarks>
+        <remarks>Although it involves no copying, accessing Span property is still costlier 
+          than reusing its return value. Try to assign its result to a variable, and use the variable
```

Comment:
```suggestion
          Instead of accessing this property multiple times, it's better to assign its result to a variable (if possible).</remarks>
```